library/Pipeline/src/Juvix/Pipeline/ToHR/Env.hs

```diff
@@ -117,6 +117,16 @@ evalEnv ::
 evalEnv ctx param env =
```

Comment:
Don't we want this applied to evalEnv and execEnv too?

Reply:
`execEnv` Already acts like this, giving back the either. As for evalEnv, since `eval<Monad>` and `exec<Monad>` have a standard meaning, I decided not to change it's typical meaning, but rather make a newer more explicit version

Reply:
I can't see why we want to ignore the errors as the default case